docs/overview.md

```diff
@@ -3,35 +3,48 @@ Spank plugins for Slurm to support quantum resources
 
 ## Content
 
+- [Context](#content)
 - [Definitions](#definitions)
   - [QPU](#qpu)
   - [Quantum computer](#quantum-computer)
+  - [IBM Quantum Platform](#ibm-quantum-platform)
+  - [Direct Access API](#direct-access-api) 
   - [Spank plugins](#spank-plugins)
   - [Spank quantum plugin](#spank-quantum-plugin)
   - [Qiskit primitives (Sampler and Estimator)](#qiskit-primitives-sampler-and-estimator)
-  - [Quantum resource for workload management systems](#quantum-resource-for-workload-management-system)
-  - [Quantum resource API](#quantum-resource-api)
+- [High Level Structure](#high-level-structure) 
+- [Quantum resource for workload management systems](#quantum-resource-for-workload-management-system)
+- [Quantum resource API](#quantum-resource-api)
 - [UX](#ux)
+  - [Backend selection as part of the configuration](#backend-selection-as-part-of-the-configuration)
   - [Running primitive jobs](#running-primitive-jobs)
   - [Running hybrid quantum-classical job with multiple resources](#running-quantum-classical-job-with-multiple-resoures)
   - [Running complex workflows](#running-complex-workflows)
-- [Integration options](#integration-options)
-  - [Gres-like](#gres-like)
-  - [Primitive jobs as separate Slurm job](#primitive-jobs-as-separate-slurm-job)
+- [Integration Flow](#integration-flow)
 - [General architecture](#general-architecture-of-plugin)
+- [Architectural Tenets](#architectural-tenents)
 
+## Context
+
+Overview of involved components, personas and backend service options:
+![context diagram](./images/context_diagram.png)
```

Comment:
We should remove or redo this and move focus to more vendor agnostic picture.

Reply:
right. Not sure if we need a vendor-specific picture... after Tuesday discussion, happy to create a separate file structure for generic/IQP/DA/other